presto-main/src/test/java/io/prestosql/sql/query/TestSessionFunctions.java

```diff
@@ -45,11 +45,11 @@ public void testCurrentPath()
         }
 
         Session emptyPathSession = testSessionBuilder()
-                .setPath(new SqlPath(Optional.of("\"\"")))
+                .setPath(new SqlPath(Optional.empty()))
                 .build();
 
         try (QueryAssertions queryAssertions = new QueryAssertions(emptyPathSession)) {
-            queryAssertions.assertQuery("SELECT CURRENT_PATH", "SELECT CAST('" + emptyPathSession.getPath().toString() + "' AS VARCHAR)");
+            queryAssertions.assertQuery("SELECT CURRENT_PATH", "VALUES VARCHAR ''");
```

Comment:
`VALUES ''` should be enough.

however, if path is not set, maybe `CURRENT_PATH` should return `NULL`?

Reply:
No, it's not. `''` has type `VARCHAR(0)`, which make the assertion fail due to type not being an exact match.